script/Contracts.sol

```diff
@@ -28,7 +29,9 @@ library MainnetContracts {
     address public constant YNBNBX = 0x32C830f5c34122C6afB8aE87ABA541B7900a2C5F;
 
     address public constant SLIS_BNB_PROVIDER = 0xfD31e1C5e5571f8E7FE318f80888C1e6da97819b;
-    address public constant YIELDNEST_MPC_WALLET = 0x24bcA21172B564474734Ae25900663BCC964d92b;
+    address public constant OLD_YIELDNEST_MPC_WALLET = 0x24bcA21172B564474734Ae25900663BCC964d92b;
```

Comment:
let's not do this rename, if there's a new wallet it will be replaced here
`YIELDNEST_MPC_WALLET`